### DISCUSSION TITLE
Allow developer to change the environment to run Tinker under (equivalent to --env option in command line)

### DISCUSSION THREAD

**Discussion by aokrasavin** [Roboroads/laravel-tinker#380] [Feature requests]
At present, when running the plugin, it always runs under default environment. It would be useful to allow use to select an environment for each console. 
Concrete example - I have a code which runs perfectly well in production, but fails during testing (which obviously runs under --dev=testing). This was ultimately traced to issues in configurations.

**Comment by Roboroads** [comment]
You can change the environment variables in your PHP Interpreter settings :)
